frontend/src/scenes/experiments/ExperimentView/CumulativeExposuresChart.tsx

```diff
@@ -0,0 +1,104 @@
+import { useValues } from 'kea'
+import { InsightEmptyState } from 'scenes/insights/EmptyStates'
+
+import { Query } from '~/queries/Query/Query'
+import { InsightVizNode, NodeKind, TrendsQuery } from '~/queries/schema'
+import { BaseMathType, ChartDisplayType, Experiment, InsightType } from '~/types'
+
+import { experimentLogic } from '../experimentLogic'
+
+const getCumulativeExposuresQuery = (experiment: Experiment): InsightVizNode<TrendsQuery> => {
+    const experimentInsightType = experiment.filters?.insight || InsightType.TRENDS
+
+    // Trends Experiment
+    if (experimentInsightType === InsightType.TRENDS) {
+        const exposureName = experiment.parameters?.custom_exposure_filter?.events?.[0]?.name
```

Comment:
When a user specifies a custom exposure filter, simply grab that filter and convert it into a query using `queryFromFilters` https://github.com/PostHog/posthog/blob/master/frontend/src/queries/nodes/InsightViz/utils.ts#L112

..and then pass the resulting query into `<Query />`

`filters` is the old schema and `query` is the new (HogQL) schema. We'll soon convert all `filters` on experiment records to `queries`, but for now we need to perform the conversion in the frontend like this.

Reply:
Applied with 141282baeb4b57fd0946efd3d39814f0752ee4b9